### PR TITLE
UnitTest

### PR DESCRIPTION
**Pull request by JananiKugaa**

### CODE REVIEW

**Review by keerthu — APPROVED** [review]

File: src/test/java/org/wso2/carbon/inbound/poll/unit/test/dbeventlistener/DBEventPollingConsumerTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    @Test
+    public void isRollbackTestWithNoRollback() throws Exception {
+        org.apache.axis2.context.MessageContext mc = new org.apache.axis2.context.MessageContext();
+        SynapseConfiguration config = new SynapseConfiguration();
+        SynapseEnvironment env = new Axis2SynapseEnvironment(config);
+        MessageContext messageContext = new Axis2MessageContext(mc, config, env);
+        Assert.assertFalse(Whitebox.invokeMethod(dbEventPollingConsumer, "isRollback", messageContext));
+    }
+}
```
Comment (menakaj) [inline src/test/java/org/wso2/carbon/inbound/poll/unit/test/dbeventlistener/DBEventPollingConsumerTest.java:114]: Add new line here.

File: src/test/resources/testng.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+<suite name="ESBTestSuite" parallel="false">
+    <test name="DBEvent-Connector-Unit-Test">
+        <classes>
+            <class name="org.wso2.carbon.inbound.poll.unit.test.dbeventlistener.DBEventPollingConsumerTest"/>
+        </classes>
+    </test>
+</suite>
+
+
```
Comment (menakaj) [inline src/test/resources/testng.xml:29]: Remove this extra line.

File: src/test/java/org/wso2/carbon/inbound/poll/unit/test/dbeventlistener/DBEventPollingConsumerTest.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        properties.setProperty(DBEventConstants.TABLE_PRIMARY_KEY, "ID");
+        properties.setProperty(DBEventConstants.DB_TABLE, "tables");
+        SynapseConfiguration config = new SynapseConfiguration();
+        SynapseEnvironment env = new Axis2SynapseEnvironment(config);
+        dbEventPollingConsumer = new DBEventPollingConsumer(properties, "test", env, 1000, "seq1", "error1", false,
+                false);
+        resultSet = Mockito.mock(ResultSet.class);
+    }
+
+    @Test
```
Comment (menakaj) [inline src/test/java/org/wso2/carbon/inbound/poll/unit/test/dbeventlistener/DBEventPollingConsumerTest.java:60]: Add descriptions for Tests.

**Comment by menakaj** [comment]
Use proper commit messages. Please find all and re-word them.

**Pull request merged**